js/anims-missiles.js

```diff
@@ -91,4 +100,23 @@ function listenKeypressMissiles() {
 
 createMissileEventHandler()
 listenKeypressMissiles()
+
+
+function incrementToll() {
+    var previousToll = parseInt(document.querySelector('#toll').innerHTML)
+    var currentToll = Math.round(previousToll + ((Math.random()+1)*1E6))
```

Comment:
what does this even do?! not very readable code, please either use 100000 or at least comment what this does

Reply:
I didn't write this. I imagine this may be a leftover from Imogen and Burak's stuff that didn't get pulled for whatever reason.